package.json

```diff
@@ -30,8 +30,8 @@
     "url": "https://github.com/tessel/node-usb.git"
   },
   "scripts": {
-    "install": "prebuild-install --verbose || node-gyp rebuild",
-    "prebuild": "prebuild --force --strip --verbose",
+    "install": "prebuild-install --runtime napi || node-gyp rebuild",
```

Comment:
Is it worth keeping the `--verbose` flag? Not sure of the benefit it brings to the user.

Reply:
I'll put it back, didn't remove it on purpose 👍